contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Standup",
+    "description": "Web app that logs daily standup updates",
+    "repository": {
+        "url": "https://github.com/mozilla/standup",
+        "license": "BSD-3-Clause",
+        "tests": "https://travis-ci.org/mozilla/standup/"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Standup",
```

Comment:
I thought about that too while prepping this PR... hmm... we could make the `/contribute.json` URL dynamic and the values set via settings, but that seems a bit extreme at this point as well... I'm not sure either. I'm happy to remove the wiki from the file though and just point at the repo as "home". Maybe that would be slightly better?